docs/book/src/developers/development.md

```diff
@@ -21,6 +21,8 @@
     - [Tilt for dev in CAPZ](#tilt-for-dev-in-capz)
     - [Tilt for dev in both CAPZ and CAPI](#tilt-for-dev-in-both-capz-and-capi)
     - [Deploying a workload cluster](#deploying-a-workload-cluster)
+    - [Tilt for development (Microsoft Tenant)](#tilt-for-development-microsoft-tenant)
```

Comment:
Rather than describe this in terms of Azure Tenant requirements, let's describe this in terms of the actual functionality. I would just describe it as "local VNET node-apiserver communication"

Reply:
Added via ceaefa853e05ad78c0b5b0e3e7b73b44229e5d58